### ISSUE TITLE
crnn模型训练

### ISSUE THREAD

**Issue by isLiuHao** [PaddlePaddle/PaddleOCR#5059]
您好，我使用的rec_r34_vd_none_bilstm_ctc.yml配置在V100上进行训练，数据集为360W的《中文文档文字识别》。训练了100个epoch，acc仍未0.006左右，请问一下这种情况正常吗？还有没有比较继续训练？

**Comment by andyjpaddle** [comment]
请提供详细的配置文件和运行指令

**Comment by isLiuHao** [comment]
> 请提供详细的配置文件和运行指令
`
Global:
  algorithm: CRNN
  use_gpu: true
  epoch_num: 1000
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: output/rec_CRNN
  save_epoch_step: 3
  eval_batch_step: 2000
  train_batch_size_per_card: 2048
  test_batch_size_per_card: 2048
  image_shape: [3, 32, 320]
  max_text_length: 25
  character_type: ch
  character_dict_path: ./ppocr/utils/lian-med-dict.txt
  loss_type: ctc
  distort: false
  use_space_char: true
  reader_yml: ./configs/rec/rec_chinese_reader.yml
  pretrain_weights:
  checkpoints:
  save_inference_dir:
  infer_img:


Architecture:
  function: ppocr.modeling.architectures.rec_model,RecModel

Backbone:
  function: ppocr.modeling.backbones.rec_resnet_vd,ResNet
  layers: 34
 
Head:
  function: ppocr.modeling.heads.rec_ctc_head,CTCPredict
  encoder_type: rnn
  SeqRNN:
    hidden_size: 256
    
Loss:
  function: ppocr.modeling.losses.rec_ctc_loss,CTCLoss

Optimizer:
  function: ppocr.optimizer,AdamDecay
  base_lr: 0.001
  beta1: 0.9
  beta2: 0.999
`
`
python3 tools/train.py \
-c configs/rec/rec_r34_vd_none_bilstm_ctc.yml \
-o Global.character_dict_path=./ppocr/utils/lian-med-dict.txt \
Global.save_model_dir=./output/rec_r34_vd_none_bilstm_ctc/ \
>./train_log/rec_r34_vd_none_bilstm_ctc.log 2>&1 &
`

**Comment by andyjpaddle** [comment]
建议使用最新版release2.4版本哦，现在主推动态图，develop版本不再维护更新了
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.4/configs/rec/rec_r34_vd_none_bilstm_ctc.yml

**Comment by isLiuHao** [comment]
> 建议使用最新版release2.4版本哦，现在主推动态图，develop版本不再维护更新了 https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.4/configs/rec/rec_r34_vd_none_bilstm_ctc.yml

2.4版本是可以直接训练静态图是吗？

**Comment by andyjpaddle** [comment]
先训练动态图，动态图方便训练调试，然后可以通过转换为静态图来进行预测部署。

**Comment by andyjpaddle** [comment]
预测推理具体可参考：https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.4/doc/doc_ch/inference.md
